crates/zksync_error/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "zksync_error"
-version = "0.3.1"
-edition = "2021"
+version.workspace = true
+edition.workspace = true
```

Comment:
This should inherit a few more things (license, keywords etc), like all the other crates in the repo

Reply:
Good point! Inherited other things too.